README.md

```diff
@@ -52,7 +52,7 @@ Note: A 'flowId' value is required for the widget to interact with PingFederate,
 This is a good choice if you already have a node project with `package.json` and can just add this widget as a dependency.
 
 To add the widget as a dependency:
-1. Run `npm install pf-authn-widget --save`
+1. Run `npm install @ping-identity/pf-authn-js-widget --save`
```

Comment:
@khosravian I believe this is the only instruction that needs updating. Let me know if there is more and I will make the change.

Reply:
O this doesn't need to change

Reply:
`authnwidget` is correct

Reply:
Oh, thanks. Let me back that out.